src/rssparser.cpp

```diff
@@ -37,15 +37,15 @@ rss_parser::rss_parser(const std::string& uri,
 	, easyhandle(0)
 {
 	is_ttrss = cfgcont->get_configvalue("urls-source") == "ttrss";
-	is_newsblur = cfgcont->get_configvalue("urls-source") == "newsblur";
+	is_NewsBlur = cfgcont->get_configvalue("urls-source") == "NewsBlur";
```

Comment:
Probably should stay as `is_newsblur`, it's a variable name. Also, `== "newsblur"`.